### PR TITLE
Add validation textarea char counter

### PR DESCRIPTION
**Pull request by debrianruhut**
Create validation on create & edit form Courses.

### CODE REVIEW

File: resources/views/layouts/master.blade.php [hunk 1]
```diff
@@ -193,5 +193,6 @@
 
 <script src="{{ asset('js/app.js') }}"></script>
 @yield('scripts')
+@stack('js')
```
Comment (angelkurten) [inline resources/views/layouts/master.blade.php]: Not is necesary, use section scripts

**Review by angelkurten — CHANGES_REQUESTED** [review]

File: resources/views/layouts/master.blade.php (hunk 1 above)
Comment (debrianruhut) [reply resources/views/layouts/master.blade.php]: Ok Angel, I'll try to fix it.

**Review by angelkurten — APPROVED** [review]

**Comment by angelkurten** [comment]
Changes in develop, thanks for contribute to LMS-Laravel

**Comment by debrianruhut** [comment]
> 
> 
> Changes in develop, thanks for contribute to LMS-Laravel

Learning by doing using github for collaboration, thank you for your guidance Angel. I'll try another pull request.

**Pull request merged**